src/controller/actions/topicDashboard.js

```diff
@@ -20,31 +20,35 @@ export const resourceError = error => ({
 export const ADD_RESOURCE = 'ADD_RESOURCE';
 export const addResource = resource => ({
   type: ADD_RESOURCE,
-  resource
+  payload: resource
 });
 
 export const UPDATE_RESOURCE = 'UPDATE_RESOURCE';
-export const updateResource = (resource, id) => ({
+export const updateResource = resource => ({
   type: UPDATE_RESOURCE,
-  resource,
-  id
+  payload: resource
 });
 
-export const DELETE_RESOURCE = 'DELETE_RESOURCES';
+export const UPDATE_RESC_ORDER = 'UPDATE_RESC_ORDER';
+export const updateRescourceOrder = rescOrder => ({
```

Comment:
Please fix this typo here and all places it's imported.

```js
-  updateRescourceOrder
+  updateResourceOrder
```

Reply:
Fixed :)